.github/workflows/release.yml

```diff
@@ -193,9 +193,7 @@ jobs:
         db: [ "mongo44", "postgres15" ]
         include:
           - db: postgres15
-            markers: "not sql"
-          - db: postgres15
-            markers: "sql"
+            markers: "and not sql"
     
```

Comment:
@alephnull this looks incorrect for me so I made some changes. 

Based on [pytest_sql.ini](https://github.com/TykTechnologies/tyk-automated-tests/blob/master/pytest_sql.ini#L2) not sql is for sql (inverse logic)

Then removed the duplication that was made on the include. Also moved the "and" inside the markers, given it will error out for the mongo44 case